.github/workflows/build_and_test.yml

```diff
@@ -23,6 +24,21 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Build dcm2niix
```

Comment:
Could this be replaced with a pip install of the pip package version? Or is there a particular reason you want to build form source? if you build from source in the MAP then do that here, but maybe an opportunity to simplify the build if you switch both to pip

Reply:
The pip version of dcm2niix is more than a year behind the source version. Previously when I was using an out-of-date dcm2niix I did actually run into some issues, so figure keeping it up-to-date is best.